tools/cleanlab/cleanlab_issue_handler.xml

```diff
@@ -0,0 +1,135 @@
+<tool id="cleanlab_issue_handler" name="Cleanlab Issue Handler" version="1.1.0">
+    <description>Detect and optionally clean data issues using Cleanlab</description>
+
+    <requirements>
+        <container type="docker">mojudy/galaxy_cleanlab:1.1.0</container>
+    </requirements>
+
+    <command detect_errors="exit_code">
+    <![CDATA[
+        python '${__tool_directory__}/cleanlab_issue_handler.py'
+        --csv '$input_file'
+        --task '$task'
+        --method '$method'
+        #if $summary_only:
+            --summary
+        #end if
+        #if not $label_issues:
+            --no-label-issues
+        #end if
+        #if not $outliers:
+            --no-outliers
+        #end if
+        #if not $near_duplicates:
+            --no-near-duplicates
+        #end if
+        #if not $non_iid:
+            --no-non-iid
+        #end if
+    ]]>
+    </command>
+
+    <inputs>
+        <param name="input_file" type="data" format="csv" label="Input CSV file (must contain 'target' column)"/>
+        <param name="task" type="select" label="Task type">
+            <option value="classification">Classification</option>
+            <option value="regression">Regression</option>
+        </param>
+        <param name="method" type="select" label="Cleaning method">
+            <option value="remove">Remove problematic rows</option>
+            <option value="replace">Replace problematic labels (classification only)</option>
+        </param>
+        <param name="summary_only" type="boolean" label="Only report issues (do not clean data)" checked="true"/>
+        <param name="label_issues" type="boolean" label="Include label issues" checked="true"/>
```

Comment:
please use argument="--no-label-issues" everywhere if possible

Reply:
```suggestion
        <param name="label_issues" type="boolean" truevalue="--no-label-issues" falsevalue="" label="Include label issues" checked="true"/>
```

In the case of a boolean you can also do this trick ... and simplify your cheetah.